drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/ListGroupItemPresenter.java

```diff
@@ -71,9 +71,9 @@ public void showAll() {
 
     @Override
     public void selectProperty(String factName, List<String> propertyParts) {
-        final ListGroupItemView listGroupItemView = listGroupItemViewMap.get(factName);
-        if (!listGroupItemView.isShown()) {
-            onToggleRowExpansion(listGroupItemView, false);
+        final ListGroupItemView instanceListGroupItemView = listGroupItemViewMap.get(factName);
+        if (!instanceListGroupItemView.isShown()) {
```

Comment:
@jomarko There is a particular reason about this rename?

Reply:
From my point of view `listGroupItemView` just says what the variable is instance off, while `instanceListGroupItemView` tries to provide more information, it is an `ListGroupItemView` representing an `Instance` , however if you think my rename is inapproprtiate I can revert it. Or if you have different proposal for name.

Reply:
@jomarko Thank you, fine for me.